platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -81,76 +81,77 @@ identity access, infrastructure operations, and app lifecycle.
-including Web portals and pages, scenario-specific code templates, automatable 
-APIs and command-line tools.
-
-According to Atlassian [[1]], "platform teams create capabilities that can be 
-used by numerous stream-aligned teams with little overhead.... platform teams 
-minimize resources and cognitive load of the stream-aligned team... platform 
-teams can create a cohesive experience that spans across different user 
-experiences or products."
-
-According to Martin Fowler and Evan Bottcher [[2]], "a digital platform is a 
-foundation of self-service APIs, tools, services, knowledge and support which 
-are arranged as a compelling internal product. Autonomous delivery teams can 
-make use of the platform to deliver product features at a higher pace, with 
-reduced coordination."
-
-Platforms are particularly relevant for cloud-native computing, which typically 
-separates supporting capabilities from application-specific logic more than 
-previous paradigms. In such environments resources like databases and object 
-stores, message queues and brokers, observability collectors and dashboards, 
-user directories and authentication systems, task runners and reconcilers and 
-more are managed independently and integrated into applications running in 
-containers and machines. A platform for cloud-native computing provides these to 
-many teams in ways that make them easy to integrate in applications and systems.
-
-At its most basic, a platform provides consistent experiences to application 
-developers for acquiring and using individual services such as a database system 
-or a secret store. Example use cases an enterprise can meet with a basic platform 
-include the following:
-
-1. Developers of products and services can automatically provision runtime 
-   capabilities such as compute, storage, databases and service identity and 
-   immediately integrate and use those as part of their products
-1. Developers of products and services can automatically provision supporting 
-   services such as task runners, package registries, deployment orchestrators and 
-   observability systems to build, verify, operate and observe their products
-1. Operators of third-party products and services can automatically provision 
-   spaces and supporting services to deploy and use those third-party products and 
-   services
-
-A more advanced platform also _composes_ these capabilities into experiences and 
-templates fit for key scenarios like web application development or data analysis. 
-For example, an application developer platform can include templates for provisioning 
-complete development environments and could serve the following use cases:
+At its more basic, a platform is an **integrated product** developed by a 
+platform engineering team that is custom built and composed of supporting 
+capabilities and services designed to reduce the cognitive load and burden 
+experienced by **internal** stakeholders of an enterprise organization 
```

Comment:
While a platform may indeed often be a product, I don't think that's _intrinsic_ to its definition. That is, you _could_ have a platform that's not a product; in fact I'd argue that we emphasize the idea of "platform as product" because it's really _not_ a product - it's an internal utility but for it to succeed it must be treated _like_ a product.

I'd prefer we stick with the original 3 paragraphs for this section.

Reply:
This is a tough one because I think that our group wants to advocate for the platform to be considered a platform while in reality that is still quite far to the right on the adoption curve.

By adding it here on the first line we are surely setting a stake in the ground which could either be very good way to build a shared assumption with the reader or could risk alienating. I am quite torn 🤔